dcgm/README.md

```diff
@@ -109,7 +116,7 @@ instances:
 
 Use the `extra_metrics` configuration field to add metrics that go beyond the ones [we support out of the box][6]. See [here][10] for the full list of metrics that dcgm-exporter can collect. Make sure to [enable these fields in the dcgm-exporter configuration][9] as well.
 
-<!-- xxx tab xxx -->
+<!-- xxz tab xxx -->
 <!-- xxx tab "Docker" xxx -->
```

Comment:
Should it be `xxz` here too? I'm not sure how that works tbh

Reply:
Yeah, I think it should be 👍